src/index.test.js

```diff
@@ -114,6 +114,17 @@ test('console.error will not be called if disableLog is set', () => {
   getConfigForFile.throwError = null
 })
 
+test('console receives output of both eslintConfig and prettierOptions when sillyLogs is set', () => {
+  format.options.sillyLogs = true
+
+  format({text: ''})
+  // TODO: fix this test, since it fails on the matcher toHaveBeenCalledTimes
```

Comment:
See this: https://github.com/kentcdodds/prettier-eslint/pull/8/files#diff-2382a7144b80bd7c6cfe69ed37c02f0cR8
And this: https://github.com/kentcdodds/prettier-eslint/pull/8/files#diff-2382a7144b80bd7c6cfe69ed37c02f0cR65